docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/snowplow-client-for-gtm-ss/snowplow-client-configuration/index.md

```diff
@@ -4,32 +4,36 @@ date: "2021-11-24"
 sidebar_position: 100
 ---
 
+:::tip
```

Comment:
This looks good! I wonder if it looks even better with the title as the tip header rather than a H2 inside the tip?

https://docusaurus.io/docs/markdown-features/admonitions#specifying-title

Reply:
It looks as a proper tip now!

![image](https://user-images.githubusercontent.com/35838196/197913017-3f13d396-216e-4650-992e-d81cefa6f533.png)